openfisca_us/variables/income/spm_unit/spm_unit_net_income.py

```diff
@@ -4,26 +4,12 @@
 class spm_unit_net_income(Variable):
     value_type = float
     entity = SPMUnit
-    label = "SPM unit net income"
+    label = "Net income"
```

Comment:
Think we probably already went over this, but just to get it down, I do support this generally - I think we should avoid putting the entity in the label.